test/test_network_constructors.jl

```diff
@@ -118,7 +118,7 @@ end
     c_sys14_dc = build_system("c_sys14_dc")
     systems = [c_sys5, c_sys14, c_sys14_dc]
     objfuncs = [GAEVF, GQEVF, GQEVF]
-    constraint_names = [:RateLimit_ub__Line, :RateLimit_lb__Line]
+    constraint_names = [:RateLimit_ub__Line, :RateLimit_lb__Line, :nodal_balance_active]
```

Comment:
This is also the name of the expression which is not strictly the same thing.

Reply:
What name do you suggest? I was just following the existing naming practices in PSI

Reply:
```
:Bus_active_power_balance
:Bus_reactive_power_balance
```